examples/maintained/make_prediction_dataflow.py

```diff
@@ -49,30 +47,60 @@
     DNNClassifierModelConfig,
 )
 
+from feature.git.dffml_feature_git.feature.operations import count_authors,git_commits,work,check_if_valid_git_repository_URL
+from feature.git.dffml_feature_git.feature.definitions import *
+
+from dffml.df.base import op
 
-"""
-TODO
-    * Insert prediction to database
-    * make dataflow_yaml general
-    * generalize??
-    * export dataflow -> predict_dataflow
-    * change documentation
-"""
 
 dataflow_yaml = "./cgi-bin/dataflow.yaml"
 
+#operation to collect features [authors,work,commits] to make features to pass to model
+@op(
+    inputs = {
+        "authors" : count_authors.op.outputs["authors"],
+        "work" : work.op.outputs["work"],
+        "commits" : git_commits.op.outputs["commits"]
+        }
+    ,
+    outputs ={
+        "features" : Definition(
+                name="maintained_features",
+                primitive="Dict[str,Any]"
+                )
+            }
+)
+async def collect_maintained_features(authors,work,commits):
```

Comment:
Great idea with this!

Reply:
This makes me think we might want to add some logic to `DataFlow._fromdict` to use `dffml.util.entrypoint:load` to load operation implementations and add them to `DataFlow.implementations` if the name is in the entrypoint format.

```yaml
  collect_maintained_features:
    inputs:
      authors: author_count
      commits: commit_count
      work: work_spread
    name: make_prediction_dataflow:collect_maintained_features
    outputs:
      features: repo_features
    stage: processing
```

That way users wouldn't have to add to the `setup.py` (or not even create one) for these kinds of one-off operations.

Reply:
> This makes me think we might want to add some logic to `DataFlow._fromdict` to use `dffml.util.entrypoint:load` to load operation implementations and add them to `DataFlow.implementations` if the name is in the entrypoint format.
> 
> ```yaml
>   collect_maintained_features:
>     inputs:
>       authors: author_count
>       commits: commit_count
>       work: work_spread
>     name: make_prediction_dataflow:collect_maintained_features
>     outputs:
>       features: repo_features
>     stage: processing
> ```
> 
> That way users wouldn't have to add to the `setup.py` (or not even create one) for these kinds of one-off operations.

Yes,i was wondering about this,i'll do that.